gensec.py

```diff
@@ -24,29 +24,36 @@
 from ase.io import write
 
```

Comment:
This file is a very large piece of code with many if/else and no docstrings or documentation. This should at least include a much much more detailed docstring that documents the workflow that is being implemented.

Reply:
Erased